wiki/FAQ/en.md

```diff
@@ -296,49 +303,58 @@ If you have any questions regarding pp, please refer to the [wiki page for pp fi
-Absolutely _nothing_. There is no difference in any way when comparing a Level 100 player against a Level 1 player except the player's own skills and experience of play. There is no handicap for the lower level nor there is an advantage for the higher level (except personal experience and hardware used).
+**Absolutely nothing**. There is no difference in any way when comparing a Level 100 player against a Level 1 player except the player's own skills and experience of the game mode in question. There is no handicap for the lower level nor is there an advantage for the higher level (except personal experience and hardware used).
 
-Level is based on your total score accumulated when online with your account whether you pass or fail the beatmap. [Formula given here][Score#Level wikilink].
+Level will be based on your total score accumulated when online with your account whether you passed or failed the beatmap. [Formula given here][Score#Level wikilink].
 
 #### Why is my profile stats different ingame?
 Keep in mind that osu! has four game modes so you might be looking at the wrong game mode.
 
-Manually select the game mode at your profile that you want to view.
-
 ![Profile_Mode_Buttons][Profile_Mode_Buttons image]
 
+_Manually select the game mode at your profile that you want to view._
+
+#### How do I change the default keyboard key input for my game mode of choice?
+![Options sidebar with quickicons][Options quickicons image]
+
+You may configure the keyboard buttons as you see fit, from the _Input_ icon in the _Options_ sidebar as shown above, under **Keyboard** section as shown below.
+
+![Options keyboard section][Options keyboard image]
+
 #### Why am I seeing random white lines on screen every time I play?
-Make sure that the smoke/skywriting key binding is not the same as the keybinding you use at playing.
+Make sure that the smoke/skywriting key binding is not the same as the key binding you use to tap with when playing.
+
+![osu! smoke effect][osu_smoke image]
 
-Image | Description
-:---: | :---
-![osu! smoke effect][osu_smoke image] | Usage of "Smoke"
-![osu! smoke settings][osu_smoke_set image] | "Smoke" in key bindings settings
+_Usage of Smoke_
 
-### Why do I not have the max possible combo when I have no misses?
-You released a slider too early or you barely clicked the hit-circle. [Answer's Source][reddit osugame wiki]
+![osu! smoke settings][osu_smoke_set image]
+
+_Smoke in key bindings settings; hold `C` key to trigger by default in osu!_
+
+<!-- ### Why do I not have the max possible combo when I have no misses? -->
+<!-- You released a slider too early or you barely clicked the hitcircle. [Answer's Source](https://www.reddit.com/r/osugame/wiki/index "Reddit r/osugame's wiki") -->
 
 ### Beatmapping
 #### "This beatmap you're trying to submit isn't owned by you!"
 _For unranked beatmaps only._
 
 ![Example of an .osu file content][Beatmap_not_owned image]
 
-Please follow these steps:
+Please follow these steps:-
```

Comment:
You might need to add an extra space before the numbered list, GitHub displays the numbered list as inline.